# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLExtensionBlendMinMax.cpp
# Commit: ccfa767dba64
# Full Hash: ccfa767dba644dc193e0246eb8e8ff3377e8b8a5
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Client-side bindings mirror for precise CC, and merge similar codepaths. r=handyman
#   
#   * Context loss using RAII
#   * Move Program reflection Client-side
#   
# ==============================================================================

diff -r 71c122ac0ca7 -r ccfa767dba64 dom/canvas/WebGLExtensionBlendMinMax.cpp
--- a/dom/canvas/WebGLExtensionBlendMinMax.cpp	Wed Jan 08 22:19:14 2020 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,26 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#include "WebGLExtensions.h"
-
-#include "GLContext.h"
-#include "mozilla/dom/WebGLRenderingContextBinding.h"
-#include "WebGLContext.h"
-
-namespace mozilla {
-
-WebGLExtensionBlendMinMax::WebGLExtensionBlendMinMax(WebGLContext* webgl)
-    : WebGLExtensionBase(webgl) {
-  MOZ_ASSERT(IsSupported(webgl), "Don't construct extension if unsupported.");
-}
-
-WebGLExtensionBlendMinMax::~WebGLExtensionBlendMinMax() {}
-
-bool WebGLExtensionBlendMinMax::IsSupported(const WebGLContext* webgl) {
-  if (webgl->IsWebGL2()) return false;
-
-  return webgl->GL()->IsSupported(gl::GLFeature::blend_minmax);
-}
-
-}  // namespace mozilla